modules/UI/UI.js

```diff
@@ -1267,7 +1269,11 @@ UI.showDeviceErrorDialog = function (micError, cameraError) {
 
     message = `${message}${doNotShowWarningAgainSection}`;
 
-    messageHandler.openDialog(
+    // To make sure we don't have multiple error dialogs open at the same time,
+    // we will just close the previous one if we are going to show a new one.
+    deviceErrorDialog && deviceErrorDialog.close();
+
+    deviceErrorDialog = messageHandler.openDialog(
```

Comment:
Please excuse my ignorance. The following question stems from the fact that the problem was that 2 dialogs would appear on top of each other. Did they report different error messages? If the first dialog reports one error and the seconds dialog reports a second different error, will the first error message be lost to the user when the second dialog is opened and the first dialog is automatically closed prior to the opening of the second dialog?